mail/gnupg2/Portfile

```
@@ -4,14 +4,14 @@ PortSystem          1.0
 
 set my_name         gnupg
 name                ${my_name}2
-version             2.2.36
+version             2.2.39
 revision            1
```

`revision` should reset to 0 on version bump.

```suggestion
revision            0
```